tests/tickets/1572/1.js

```diff
@@ -0,0 +1,111 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: wysiwygarea,toolbar,forms,widget,codesnippet,undo,basicstyles,entities */
+
+'use strict';
+( function() {
+
+	var backspace = 8;
+	var deleteKey = 46;
+	var leftArrow = 37;
+	var rightArrow = 39;
+
+	var input = (
+		'<pre><code class="language-javascript">//Hello world!</code></pre><p>&nbsp;</p><pre>' +
+		'<code class="language-javascript">//Hello another world!</code></pre>'
+	);
+
+	var notEmptyInput = (
+		'<pre><code class="language-javascript">//Hello world!</code></pre><p>Hello, world!</p><pre>' +
+		'<code class="language-javascript">//Hello another world!</code></pre>'
+	);
+
+
+	var result = (
+		'<pre><code class="language-javascript">//Hello world!</code></pre><pre>' +
+		'<code class="language-javascript">//Hello another world!</code></pre>'
+	);
+
+	function selectEmptyParagraph( editor ) {
+		var range = editor.createRange();
+		range.selectNodeContents( editor.document.getElementsByTag( 'P' ).getItem( 0 ) );
+		range.collapse( true );
+		editor.getSelection().selectRanges( [ range ] );
+	}
+
+	bender.test( {
+		'test fire delete to remove an empty paragraph between two widgets': function() {
+			bender.editorBot.create( {
+				name: 'editor1',
+				startupData: input
+			}, function( bot ) {
+				var editor = bot.editor;
+				selectEmptyParagraph( editor );
+				editor.editable().fire( 'keydown', new CKEDITOR.dom.event( { keyCode: deleteKey } ) );
+				assert.areSame( result, editor.getData(), 'Empty paragraph between two widget should be removed.' );
```

Comment:
There should be an empty line before assertions block (for all `assert` statements in this file).